src/inputs/style.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import makeStyles from '@mui/styles/makeStyles';
+
+const useStyles = makeStyles((theme) => ({
+  dirtyInput: {
+    paddingLeft: '16px',
+    borderLeft: `${theme.palette.info.main} solid 3px`,
+  },
+  notDirtyInput: {
+    paddingLeft: '19px',
+  },
+  toolBar: {
+    height: '40px',
+    display: 'flex',
+    flexDirection: 'row',
+    flexFlow: 'flex-start',
+    alignItems: 'stretch',
+    marginTop: '10px',
+    marginBottom: '6px;',
+  },
+  errorsContainer: {
+    backgroundColor: theme.palette.black,
+    color: theme.palette.text.error,
+    fontSize: '15px',
+    weight: 600,
+    marginTop: '10px',
+    padding: '4px',
+    whiteSpace: 'pre-line',
+  },
+  loadingLabel: {
+    marginLeft: '15px',
+    marginRight: '15px',
+    marginTop: 'auto',
+    marginBottom: 'auto',
+  },
+  loadingSpinner: {
+    marginTop: 'auto',
+    marginBottom: 'auto',
+  },
+}));
```

Comment:
I had to put the table part here too since style is generated with the `makeStyles` function. I understand I should only create a dict that contains CSS for every input and use `makeStyles` in each input separately, I just don't know the syntax yet.

Reply:
Merci @csm-thu pour le fix!